live-examples/js-examples/string/string-substring.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js">const str = 'Mozilla';
+console.log(str.substring(1,2)); // 'oz'
+console.log(str.substring(2,str.length)); 'zilla'
+</code>
+</pre>
+  
```

Comment:
Please remove this line containing just spaces, ad all files should end with a return character (that's what the 
<img width="30" alt="screen shot 2018-08-10 at 12 22 07 pm" src="https://user-images.githubusercontent.com/432915/43977131-fc2fbfd2-9c97-11e8-87b9-ab97a37e541a.png"> is telling you).

Reply:
Done